boot/boot.go

```diff
@@ -40,6 +42,43 @@ const (
 	TryingStatus = "trying"
 )
 
+var (
```

Comment:
maybe these should got to their own file initramfs20dirs.go ?  can be in a follow up

Reply:
sure I will put these in their own file in a followup